lua/harpoon/buffer.lua

```diff
@@ -8,7 +8,7 @@ local HARPOON_MENU = "__harpoon-menu__"
 -- simple reason here is that if we are deving harpoon, we will create several
 -- ui objects, each with their own buffer, which will cause the name to be
 -- duplicated and then we will get a vim error on nvim_buf_set_name
-local harpoon_menu_id = math.random(1000000)
+local harpoon_menu_id = math.random(100000)
```

Comment:
For experience, I'm thinking you can revert only this part of code to no cause unnecessary conflicts with your other PR https://github.com/ThePrimeagen/harpoon/pull/414/files

Reply:
Yep thanks for reminding me! Forgot to do this when I made the other PR :)